### ISSUE TITLE
webserver

### ISSUE THREAD

**Issue by martin-raden** [fabriziocosta/RaSE#8]
- [ ] FORNA visualization of mfe
- [ ] scrollable full table output

**Comment by fabriziocosta** [comment]
how is it coming along?

**Comment by martin-raden** [comment]
Milad set it up (incl deps) on the cluster.. will have a look tomorrow if working. if so, i start setting up the server...

any parameters to exclude from the webinterface?

**Comment by martin-raden** [comment]
let me know what else your need or what can be improved:

[example output](http://rna.informatik.uni-freiburg.de/trunk/RetrieveResults.jsp?jobID=0078358&toolName=RaSE)

**Comment by fabriziocosta** [comment]
cool. 
we could have the table a bit taller and with scrolling, i.e. having all data available to be selected instead of having it accessible clicking on different links.

**Comment by fabriziocosta** [comment]
Bit number for representation -> Bit size for feature encoding

**Comment by fabriziocosta** [comment]
Mark best set to 6 by default

**Comment by martin-raden** [comment]
param name and default number is fixed

concerning the table: currently, I dont have a table setup that will enable scrolling but keeps the header visible on top. but I think this is essential for longer tables. Thus I would like to keep the current table setup, to keep the workload low.

**Comment by martin-raden** [comment]
- [ ] FORNA visualization of mfe

**Comment by mmiladi** [comment]
@martin-mann Does forna support still non-interactive images?

**Comment by martin-raden** [comment]
yes, you can switch interactivity off (done in the AntaRNA webserver)
